enhancements/proxy/global-cluster-egress-proxy.md

```diff
@@ -150,7 +150,7 @@ configmap after confirming the validation endpoints can be accessed using the ne
 
 Additional behaviors:
 
-* configmaps labeled with `config.openshift.io/inject-trusted-cabundle: "true"` will have the current
+* configmaps labeled with `config.openshift.io/inject-proxy-cabundle: "true"` will have the current
```

Comment:
https://github.com/openshift/cloud-credential-operator/blob/1ab2ca2f796cf1e77969164a67502fe910468b07/manifests/05_deployment.yaml#L141

hmm, i think the previous one is correct..?

Reply:
> hmm, i think the previous one is correct..?

I'm suggesting we [pivot][1], but haven't responded to Ben's points about [migration difficulty][2] yet.

[1]: https://github.com/openshift/enhancements/pull/115#issuecomment-554869921
[2]: https://github.com/openshift/enhancements/pull/115#issuecomment-555150622

Reply:
My  2 cents here, the new naming avoid misunderstandings as it states the scope for the ConfigMap's content.